set.mm

```diff
@@ -382312,20 +382312,23 @@ proved using the inner form (see theorem 9.6 of [Schwabhauser] p. 69
       tgbtwnxfr.e $e |- ( ph -> E e. P ) $.
       tgbtwnxfr.f $e |- ( ph -> F e. P ) $.
       tgbtwnxfr.2 $e |- ( ph -> <" A B C "> .~ <" D E F "> ) $.
-      $( Deduce segment congruence from a triangle congruence.  (Contributed by
-         Thierry Arnoux, 27-Apr-2019.) $)
+      $( Deduce segment congruence from a triangle congruence.  This is a
+         segment portion of congruent triangles are congruent (CPCTC).
```

Comment:
The second sentence seems to lack some words or have an extra verb ?
What does CPCTC stand for ?

Reply:
You're right. Sorry about that. I will tweak it later.

Reply:
I've tweaked it, hopefully that resolves the problem!